GraphStructure/Graph.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Iterator;
+
+public class Graph{
+	public ArrayList<Vertex> vertexes;
+	public ArrayList<Boolean> visitMark;
+
+	public Graph(){
+		this.vertexes=new ArrayList<Vertex>();
+		this.visitMark=new ArrayList<Boolean>();
+	}
+	public Graph(Integer graphSize){
```

Comment:
Check out the space: should be public Graph(Integer graphSize) {}. Also, it may be better to use int instead of Integer. Integer may only exist in set objects (array, list, hashtable etc.). BTW, why you need to specify the graph size? Hard code the vertex in advance is not a good way, you can always use a HashMap<id, Vertex> to maintain the vertex info, and dynamically add the vertices while adding the edges.